releasetasks/templates/release_graph.yml.tmpl

```diff
@@ -65,6 +65,19 @@ tasks:
     {% endif %}
 
     {% if verifyConfigs %}
+        {% macro updateVerify_task(platform) %}
+            {% if platform in ["linux", "linux64"] %}
```

Comment:
bear with me as I'm a noob with this but I thought we call release_graph.yml.tmpl just once and pass it all platforms? So doesn't this need to be a for loop?